scripts/dist-tag.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+# dist-tag.sh - manage dist-tags for NPM registry packages 
```

Comment:
elaborate a bit on "manage"?

I'm not really fluent in bash. I could use a comment for each clump of lines, actually.